lib/modules/dosomething/dosomething_organ_donation/dosomething_organ_donation.module

```diff
@@ -53,12 +61,34 @@ function _dosomething_organ_donation_build_http_client($nid) {
     'base_url' => $base_url,
     'headers' => [
       'Authorization' => $token,
+      'Content-Type' => 'application/json',
     ],
   ];
 
   return $client;
 }
 
+/**
+ * POST user's information to Organize's Registration API endpoint.
+ *
+ * @param array $values
+ *   Values from the registration form.
+ */
+function dosomething_organ_donation_post_registration($values) {
+  $client = _dosomething_organ_donation_build_http_client();
+
+  $options = [
+    'method' => 'POST',
+    'headers' => $client['headers'],
+    'data' =>
+      json_encode($values),
```

Comment:
wow! much cleaner.
let's also add the `source` value back in to this array, as I think that was hardcoded